cresponse/cresponse.go

```diff
@@ -20,11 +20,11 @@ func (err CError) Error() string {
 	return fmt.Sprintf("System error number %d: %s", err.number, err.description)
 }
 
-// GetGetResponsesErrors returns the Go representation of the errors present in a C.reponse_t.
+// GetResponsesErrors returns the Go representation of the errors present in a C.response_t.
 //
 // NOTE: cgo types are private to each module, so exporting a function that expects a *C.response_t doesn't make sense,
 // so we export a function with an unsafe.Pointer and we cast it internally.
-func GetResponsesErrors(responsePointer unsafe.Pointer) (harderror error, softerrors []error) {
+func GetResponsesErrors(responsePointer unsafe.Pointer) (softerrors []error, harderror error) {
```

Comment:
I see you changed the order of the returned arguments in a number of places. Any reason why?